### ISSUE TITLE
Feature: Track Habits for Days in the Past

### ISSUE THREAD

**Issue by pfirsich** [NerdyPepper/dijo#37]
After to starting to use dijo yesterday, I wanted to enter some data from the past days, but I couldn't find a way to do that. 

Personally I can also clearly imagine wanting to track a habit a day or two after, because I were busy or didn't have my computer available or something, so I think this is rather necessary.

**Comment by fredrb** [comment]
This is a very important feature for me too. I track activities in other application and I wanted to automatically import my history. 

I noticed that the data for the app is stored in `$HOME/Library/Application\ Support/rs.nerdypepper.dijo/habit_record.json` (OSx). As a workaround you could append to this file. But I agree there should be a standard way to inject data here.

```json
[
  {
    "type": "Count",
    "name": "my-project-example",
    "stats": {
      "2020-07-21": 1
    },
    "goal": 3,
    "auto": false
  }
]
```

**Comment by NerdyPepper** [comment]
This is definitely on the list of things to add, and it has been requested over at #10. I would prefer it if feature requests are documented in that thread, it makes things a lot easier for me to track. You may choose to receive notifications only if you are mentioned.

**Comment by NerdyPepper** [comment]
Closed by latest release (v0.2.6).
